Install.php

```diff
@@ -0,0 +1,76 @@
+<?php 
+
+class PHPCompatibility_Install {
+    static function enable() {
+        echo "Enabling PHPCompatibility\n";
+        self::make_copy();
+        self::register_in_cs();
+    }
+    
+    static function disable() {
+        echo "Disabling PHPCompatibility\n";
+        self::remove_copy();
+        self::unregister_from_cs();
+    }
+    
+    static function register_in_cs() {
+        if (PHP_CodeSniffer::getConfigData('installed_paths') === NULL) {
+            PHP_CodeSniffer::setConfigData('installed_paths', __DIR__);
+            echo "Registered our path in PHP CodeSniffer\n";
+        } else if (PHP_CodeSniffer::getConfigData('installed_paths') === __DIR__ ) {
+            echo "Our path is already registered in PHP CodeSniffer\n";;
+        } else {
+            throw new Exception("Another path is registered in Code Sniffer conf :(");
+        }
+    }
+    
+    static function unregister_from_cs() {
+        if (PHP_CodeSniffer::getConfigData('installed_paths') === __DIR__) {
+            PHP_CodeSniffer::setConfigData('installed_paths', NULL);
+            echo "Unregistered our path in PHP CodeSniffer\n";
+        } else {
+            echo "No path registered in Code Sniffer\n";
+        }
+    }
+
+    static function make_copy() {
+        $srcDir = __DIR__;
+        $copy = __DIR__ .DIRECTORY_SEPARATOR.'PHPCompatibility';
+        
+        if ( file_exists ($copy)) {
+            echo "Copy hack is already in place\n";
```

Comment:
> I continue to think that the right way is to make composer way the main moving permanently into a subdir.

For Composer, I agree, however, originally PHPCS was (and still is) a PEAR project, so that solution disregards any non-Composer users where Composer is the "new kid on the block".